### PR TITLE
Hyperlink ui support

### PR DESCRIPTION
**Pull request by averrin**
add https://atom.io/packages/hyperclick support

### CODE REVIEW

**Comment by xndcn** [comment]
@averrin Awesome, thank you very much!

File: lib/symbols-tree-view.coffee [hunk 1]
```diff
@@ -76,6 +76,14 @@ module.exports =
         tag = @parser.getNearestTag(row)
         @treeView.select(tag)
 
+    focusClickedTag: (editor, text) ->
+      console.log "clicked: #{text}"
+      if editor = @getEditor()
+        tag =  (t for t in @parser.tags when t.name is text)[0]
+        @treeView.select(tag)
+        # imho, its a bad idea =(
+        jQuery('.list-item.list-selectable-item.selected').click()
```
Comment (xndcn) [inline lib/symbols-tree-view.coffee]: Do you try what happened if this line deleted?

Comment (averrin) [inline lib/symbols-tree-view.coffee]: Tap will be only highlighted in tree view, but we also need move text edition to it. I think,  better way for it is emit some event, or some think more native. But im not so good understood your code.

**Comment by xndcn** [comment]
@averrin Sorry, I'm too busy to forget this... My fault, sorry... Thank you very much for your work!

**Pull request merged**